tests/Microsoft.Bot.Builder.AI.Luis.TestUtils/Utils.cs

```diff
@@ -81,7 +81,7 @@ public static bool WithinDelta(JToken expectedToken, JToken actualToken, double
                 {
                     if (expectedToken.Type == JTokenType.String)
                     {
-                        withinDelta = expectedToken.Value<string>().Equals(actualToken.Value<string>(), StringComparison.InvariantCultureIgnoreCase);
+                        withinDelta = expectedToken.Value<string>().Equals(actualToken.Value<string>(), StringComparison.Ordinal);
```

Comment:
This should be OrdinalIgnoreCase as a replacement of InvariantCultureIgnoreCase

Reply:
Fixed. Thanks!